ccmlib/utils/download.py

```diff
@@ -172,7 +172,8 @@ def get_url_hash(url: str) -> str:
     """
 
     if os.path.exists(url):  # if file/dir is local, hash based on the path
-        return hashlib.md5(url).hexdigest()
+        # Strings must be encoded before hashing
+        return hashlib.md5(url.encode('utf-8')).hexdigest()
```

Comment:
it might worth adding a unit test for this case (we have test for downloading, but not for using local file)

Reply:
added